blocks/init/components/button/components/button-editor.js

```diff
@@ -0,0 +1,29 @@
+import { Fragment } from '@wordpress/element';
+
+export const ButtonEditor = (props) => {
+  const {
+    blockClass,
+    title,
```

Comment:
Maybe rename the `title` attribute to `text` attribute.
This is usually called button text, not button title.